Modules/Server/Sources/PocketCastsServer/Private/MimetypeHelper.swift

```diff
@@ -0,0 +1,15 @@
+import Swime
+import Foundation
+
+public struct MimetypeHelper {
+    public static func contetType(for url: URL) -> String? {
+        do {
+            let data = try Data(contentsOf: url)
```

Comment:
This call will pull all the file data into memory... Is there a way to just get the first bytes?

Reply:
Yes in theory I can use the FileHandle to get the data for n amount of bytes. I need to make some change to the package anyway, I will try there